modules/frontend/auth/data/src/services/authService.ts

```diff
@@ -74,6 +77,47 @@ export function AuthService() {
     }
   };
 
+  const sendRecoverPasswordEmail = async (email: SendEmailDto) => {
+    try {
+      const response = await serviceInstance.post(`/send-recover-email`, email);
+      return {
+        token: response.data.token,
+        error: false,
+        message: response.data.message,
+      };
+    } catch (error) {
+      if (axios.isAxiosError(error) && error.response?.status && error.response?.data?.message) {
+        handleAxiosError('sendRecoverPasswordEmail', error);
+        return {
+          error: true,
+          message: error.response.data.message,
+        };
+      }
+
+      throw log.unexpectedError('sendRecoverPasswordEmail', error as Error);
+    }
+  };
+
+  const confirmPasswordByToken = async (token: string, data: ConfirmPasswordDto): Promise<DefaultResponseDto> => {
```

Comment:
i think for this caso you can use a object 

```suggestion
  const confirmPasswordByToken = async ({token, data} : {token: string, data: ConfirmPasswordDto}): Promise<DefaultResponseDto> => {
```

What do you think?

Reply:
tinha pensado nisso, mas achei que teria que incluir o token no ConfirmPasswordDto

Reply:
Done.